packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/GroupForm.tsx

```diff
@@ -96,12 +97,21 @@ const GroupForm = ({
   formType,
   onSubmit,
   initialValues = {},
-  onDelete = () => {},
+  onDelete = () => {}
 }: GroupFormProps) => {
   const navigate = useCommonNavigate();
   const { data: topics } = useFetchTopicsQuery({
-    chainId: app.activeChainId(),
+    chainId: app.activeChainId()
   });
+
+  const { data: groups = [] } = useFetchGroupsQuery({
+    chainId: app.activeChainId()
+  });
+
+  const takenGroupNames = groups.map(({ name }) => name.toLowerCase());
+
+  const [isNameTaken, setIsNameTaken] = useState(false);
```

Comment:
I think this should be in the group name uniqueness pr.

Reply:
It is done in that PR as well: https://github.com/hicommonwealth/commonwealth/pull/5454/files#diff-4c2f41ca45be294fdb637836edeaf7b058571665e4cd00325fa332d288c18ccf

Reply:
I know this is already done, but for future reference, I think that this kind of state could be simpler with `useMemo` instead– having it check if `takenGroupNames` contains the new group name.